# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini
# Commit: 86e9449aba35
# Full Hash: 86e9449aba35c7439e8a41a2804388994462c00d
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r c7d1d0cfaa6c -r 86e9449aba35 testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini
--- a/testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini	Wed Jun 22 10:30:03 2022 +0000
+++ b/testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini	Wed Jun 22 10:42:34 2022 +0000
@@ -1,15 +1,3 @@
 [contain-valid.html]
-  [e.style['contain'\] = "style" should set the property value]
-    expected: FAIL
-
-  [e.style['contain'\] = "paint style" should set the property value]
-    expected: FAIL
-
-  [e.style['contain'\] = "layout style paint" should set the property value]
-    expected: FAIL
-
-  [e.style['contain'\] = "layout paint style size" should set the property value]
-    expected: FAIL
-
   [e.style['contain'\] = "layout inline-size" should set the property value]
     expected: FAIL